dashboards/google-backupdr/backupdr-monitoring.json

```diff
@@ -0,0 +1,157 @@
+{
+  "displayName": "Backup and DR Monitoring",
```

Comment:
Let's have this match the display name in the yaml file.
```suggestion
  "displayName": "Backup/Recovery storage pool utilization",
```

Reply:
Done. Updated yaml file with the same display name